resources/devices/mobile-phone/google.json

```diff
@@ -64,45 +64,6 @@
             "isTablet": "false"
         }
     },
-    "LG Nexus 4": {
```

Comment:
I just moved these to their respectively named file to make them easier to find.